torch/csrc/Exceptions.h

```diff
@@ -138,4 +157,27 @@ struct ValueError : public PyTorchError {
   }
 };
 
+// ATen warning handler for Python
```

Comment:
I see this code was moved. Any substantive changes inside?

Reply:
Yes only the comment was moved. The rest of the code is new !